api/specs.proto

```diff
@@ -234,6 +246,14 @@ message NetworkSpec {
 	bool internal = 4;
 
 	IPAMOptions ipam = 5 [(gogoproto.customname) = "IPAM"];
+
+	// AllowContainers allows individual containers to be attached
+	// to this network. With this flag enabled individual
+	// containers running in an worker node in the cluster can
+	// attach to this network and access the services attached to
+	// this network. If this flag is not enabled(default case)
+	// independent containers cannot attach to this network.
+	bool allow_containers = 6;
```

Comment:
@diogomonica That would be mis-interpreted as the network being a public network. It is still a private network. 

I think I am fine with `ManuallyAttachable` but it may be a little awkward in the cli. But if there are no better options I am going to go ahead with that.